### DISCUSSION TITLE
Missing "And" keyword

### DISCUSSION THREAD

**Discussion by EricaOkuda-br** [badeball/cypress-cucumber-preprocessor#933] [General]
Missing "And" keyword
When using And, I get the return in cypress.
Does this happen to anyone else?
Cypress version: 12.3.0

![image](https://user-images.githubusercontent.com/96707666/214888080-ffe81b14-420a-4e04-8ef9-2e2ca7b07c9d.png)

**Comment by CiceroAndrade** [comment]
Found your answer in https://github.com/badeball/cypress-cucumber-preprocessor/issues/821
